src/config.js

```diff
@@ -230,8 +230,8 @@ global.config = {
   },
 
   terminal: {
-    minEnergyAmount: 80000,
-    maxEnergyAmount: 120000,
+    minEnergyAmount: 40000,
+    maxEnergyAmount: 50000,
```

Comment:
Why?

Reply:
my terminal we full with energy at 80k+ and i had no space

Reply:
Mind to write this as a comment to the line, so that we know in the future why we went in this direction?